doc/nrf/device_guides/working_with_nrf/nrf70/developing/stack_combo.csv

```diff
@@ -0,0 +1,9 @@
+Stack components,Wi-Fi SSID scanning with cellular backhaul,Wi-Fi application for end-device-to-cloud products,Wi-Fi + BLE application for end-device-to-cloud products,Matter over Wi-Fi products,Matter over Thread products,Dual-stack capable Matter devices (not gateways)
+Wi-Fi scan only driver,X,,,,,
+Wi-Fi client stack ,,X,X,X,,X
+Bluetooth LE stack ,,,X,,,
+Matter stack ,,,,X,X,X
+Thread stack ,,,,,X,X
+LTE modem library ,,,,,,
```

Comment:
Don't you need the LTE modem library for scan-over-LTE-backhaul?

Reply:
Scan over LTE? This list only has `Wi-Fi scan only` that can be used with anything.

Reply:
Ah the first column heading itsefl has `cellular`, so, yes, checked the LTE stack for that.